monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md

```diff
@@ -53,41 +54,41 @@ La palabra Monero es del lenguaje Esperanto. Los creadores decidieron usar Esper
 
 **Trayectoria de la recompensa por bloque de Monero:** La recompensa gradualmente caerá hasta que inicie la emisión definitiva (tail emission) a finales de mayo del 2022. En este punto, las recompensas estarán fijas en 0.6 XMR por bloque. La emisión definitiva proveerá con un continuo e indefinido incentivo para minar. Adicional a esto y quizás lo más importante, la emisión definitiva proveerá a Monero con una estable y predecible inflación incorporada, esencial para el dinero real, dinero estable (sound money).
 
-**El Laboratorio de Investigación de Monero:** Monero no sólo está comprometido en hacer una moneda fungible, sino también en la continua investigación en el campo de la privacidad financiera que involucra a las criptomonedas en general. Con ese fin, matemáticos y físicos en computación han publicado **[libros blancos](https://lab.getmonero.org/)** y tienen muchas otras **[metas de investigación](https://getmonero.org/es/design-goals/)** en las que están trabajando.
+**El Laboratorio de Investigación de Monero:** Monero no sólo está comprometido en hacer una moneda fungible, sino también en la continua investigación en el campo de la privacidad financiera que involucra a las criptomonedas en general. Con ese fin, investigadores académicos en campos de matemáticas, física, seguridad y computación en blockchain han publicado **[libros blancos](https://lab.getmonero.org/)** y tienen muchas otras **[metas de investigación](https://getmonero.org/es/design-goals/)** en las que están trabajando.
 
 **La minería es accesible:** Cualquiera que pueda conectarse a un dispositivo o navegador web puede
 participar.
 
 ### IMPLICACIONES Y USOS EN LA VIDA REAL
 
-Dado que Monero es seguro, de comisiones bajas y sin fronteras, las personas pueden enviar dinero a pesar de estados o bancos corruptos y/o quebrados. Esto empodera económicamente al individuo de **[países opresores](https://www.reddit.com/r/Monero/comments/6wczty/how_monero_changed_my_life/)** o economías deprimidas.
+Dado que Monero es seguro, de comisiones bajas y sin fronteras, las personas pueden enviar dinero a pesar de estados o bancos corruptos y/o quebrados, y se pueden conducir negocios sin competidores husmeando información crítica. Esto empodera económicamente al individuo y negocios en **[países opresores](https://www.reddit.com/r/Monero/comments/6wczty/how_monero_changed_my_life/),** economías deprimidas o ambientes de negocio altamente competitivos.
```

Comment:
"Critical information" can also be translated as "información sensible". Which one would you think could fit better the translation?

Reply:
I think it fits a bit better, I'll be changing it tomorrow as soon as I can.

Reply:
ok great :).